federation/pkg/kubefed/init/init.go

```diff
@@ -586,6 +601,28 @@ func createControllerManager(clientset *client.Clientset, namespace, name, svcNa
 	return clientset.Extensions().Deployments(namespace).Create(dep)
 }
 
+func createFedConfigMap(config util.AdminConfig, fedName, kubeconfig, dnsZoneName string, dryRun bool) (*v1.ConfigMap, error) {
+	fedClientSet, err := config.FederationClientset(fedName, kubeconfig)
```

Comment:
actually that is what I was doing earlier, but while updating code for your comments on the other PR, I completely forgot about this PR.
Will update!